packages/x/templates/blueprint/ui/crud2/config/routes.tsx.tpl

```diff
@@ -2,16 +2,16 @@
 import { IRoute } from "@bluelibs/x-ui";
 import * as React from "react";
 {{# if (hasFeature "list") }}
-import { {{ generateComponentName "list" }} } from "../components/List/{{ collectionName }}List";
+const {{ generateComponentName "list" }} = React.lazy(() => import("../components/List/{{ collectionName }}List"));
```

Comment:
we should have this opt-in not defaulted.

Reply:
the option will be on blueprint level right ? if so on specific collections or all collections